packages/seacas/libraries/ioss/src/main/io_shell.C

```diff
@@ -72,27 +74,32 @@ int main(int argc, char *argv[])
   std::string out_file = interFace.outputFile;
 
   if (rank == 0 && !interFace.quiet) {
-    fmt::print(stderr,
+    fmt::print(Ioss::DEBUG(),
```

Comment:
The `Ioss::DEBUG()` should either stay as `stderr` in this file (not a library, it is an application), or probably should be `Ioss::OUTPUT()`